.github/workflows/pull_requests.yml

```diff
@@ -8,6 +8,9 @@ env:
 
 jobs:
   Labeler:
+    permissions:
+      contents: read  # for actions/labeler to determine modified files
+      pull-requests: write  # for actions/labeler to add labels to PRs
```

Comment:
The `actions/labeler` action is using a custom secret in this case, so we don't need to give it pull request write permissions.

```suggestion
```